tests/cfgs/default/result/1kxun.pcap.out

```diff
@@ -45,9 +45,8 @@ MpegDash	1	299	1
 Line	30	19034	3
 
 Safe                           124 28754         9            
-Acceptable                     622 514902        119          
+Acceptable                     627 516099        121          
 Fun                            948 1976493       53           
-Dangerous                        5 1197          2            
```

Comment:
I think that might be the reason that the idea of merging smb1 and smb23 doesn't work...
We don't want this regression.
However, nDPI has only the concept of "breed per protocol", no "breed per flow", i.e. we cant' have right now two different flows with the same classification but with different breeds...